dynamic-layers/meta-openembedded/networkmanager/files/networkmanager.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+# Copyright (c) 2008 Saleem Abdulrasool <compnerd@compnerd.org>
+# Distributed under the terms of the GNU General Purpose License v2
```

Comment:
We need to update the license when DISTRO_FEATURES includes openrc to have GPL-2.0-only

Reply:
I removed that part if that's okay with you.

Reply:
We can't just drop the license that the original author put on this init script.  You can see the same update to LICENSE being done here, https://github.com/jsbronder/meta-openrc/blob/master/recipes-appends/oe-core/nfs-utils_%25.bbappend#L25.

Reply:
Ah, okay. Missed that. Done now. 😃